chaco/grid.py

```diff
@@ -407,8 +407,6 @@ def _orientation_changed(self):
 
 
     ### Persistence ###########################################################
-    #_pickles = ("orientation", "line_color", "line_style", "line_weight",
-    #            "grid_interval", "mapper")
 
     def __getstate__(self):
```

Comment:
I don't know if the `__getstate__` methods can also be thrown out if the serialization code is being thrown out.